src/js/components/Chart.vue

```diff
@@ -0,0 +1,334 @@
+          </td>
+        </tr>
+        <tr>
+          <th scope="row" data-cell="type income">Operating Income</th>
+          <td data-cell="numeric">
+            {{ formatUSD(inputData.operating_income_loss_y1) }}
+          </td>
+          <td data-cell="numeric">
+            {{ formatUSD(inputData.operating_income_loss) }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import { formatCurrency } from '@/js/utils';
+
+export default {
+  name: 'Chart',
+  props: {
+    inputData: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    period() {
+      const str = this.inputData.period_id.toString();
+      return `${str.substring(0, 4)} Q${str.slice(-1)}`;
+    },
+    period_y1() {
+      const str = this.inputData.period_id.toString();
+      const year = (parseInt(str.substring(0, 4), 10) - 1).toString();
+      return `${year} Q${str.slice(-1)}`;
+    },
+    X0() {
+      return 5;
+    },
+    Y0() {
+      return 5;
+    },
+    WIDTH() {
+      return 400;
+    },
+    HEIGHT() {
+      return 300;
+    },
+    rev_delta() {
+      return this.inputData.total_revenue - this.inputData.total_revenue_y1;
+    },
+    cogs_delta() {
+      return (
+        this.inputData.total_cost_of_revenue -
+        this.inputData.total_cost_of_revenue_y1
+      );
+    },
+    sga_delta() {
+      return (
+        this.inputData.total_operating_expenses -
+        this.inputData.total_operating_expenses_y1
+      );
+    },
+    cogs_pct_0() {
+      return (
+        this.inputData.total_cost_of_revenue_y1 /
+        this.inputData.total_revenue_y1
+      );
+    },
+    sga_pct_0() {
+      return (
+        this.inputData.total_operating_expenses_y1 /
+        this.inputData.total_revenue_y1
+      );
+    },
+    cogs_pct_delta() {
+      return this.cogs_delta / this.rev_delta;
+    },
+    sga_pct_delta() {
+      return this.sga_delta / this.rev_delta;
+    },
+    h_cogs_0() {
+      return this.HEIGHT * this.cogs_pct_0;
+    },
+    h_sga_0() {
+      return this.HEIGHT * this.sga_pct_0;
+    },
+    h_cogs_delta() {
+      return this.HEIGHT * this.cogs_pct_delta;
+    },
+    h_sga_delta() {
+      return this.HEIGHT * this.sga_pct_delta;
+    },
+    w_delta() {
+      return this.inputData.total_revenue / this.inputData.total_revenue_y1 - 1;
+    },
+    expenses_0() {
+      return this.h_cogs_0 + this.h_sga_0;
+    },
+    expenses_delta() {
+      return this.h_cogs_delta + this.h_sga_delta;
+    },
+    scale() {
+      return (
+        this.HEIGHT /
+        Math.max(this.expenses_0, this.expenses_delta, this.HEIGHT)
+      );
+    },
+  },
+  methods: {
+    formatUSD(input) {
+      return formatCurrency(input);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.chart-container {
+  display: grid;
+  gap: var(--shim);
+  grid-template-columns: min-content 2px auto;
+  margin: var(--double-gutter) var(--gutter);
+}
```

Comment:
@dvdherron I am not sure what to suggest yet, but there is not much indication that the chart is getting cut off as opposed to my browser window not being wide enough. Partly, their is the space between the chart on the right edge, and there isn't a scroll bar so I have no reason to think anything is being hidden (the svg has overflow hidden which is why it just clips).

Reply:
one of the best things about SVG is that they scale. I think we should take advantage of that to make sure it doesn't ever get cut off?

Reply:
I think I've adjusted these so that they scale. And I think they are no longer being clipped. I think we will have to adjust how the rounded corners are being placed on some of the charts.